src/_includes/layouts/post.njk

```diff
@@ -1,8 +1,14 @@
 ---
 layout: layouts/base.njk
 type: post
+eleventyComputed:
+    permalink: "{% if date | isFuturePost %}false{% else %}{{permalink}}{% endif %}"
 ---
-
+{% if date | shortDate > now | shortDate %}
```

Comment:
What's the last `shortDate` for? It's a bit confusing for me. This whole section implies the post is rendered and served by the server, which shouldn't be the case IMO.